src/configs/repositories/configRepository.ts

```diff
@@ -282,10 +282,13 @@ export class ConfigRepository {
    */
   public async getConfigs(
     searchParams: ConfigSearchParams,
-    paginationParams: SqlPaginationParams = { limit: 1, offset: 0 }
+    paginationParams: SqlPaginationParams = { limit: 1, offset: 0 },
```

Comment:
Don't we want to allow more results to be returned as a default? Its `getConfigs` (plural).

Reply:
the default is 10 in the openapi